### ISSUE TITLE
RailsAdmin::Extensions::PaperTrail::VersionProxy::AuditingAdapter can't modify frozen Hash

### ISSUE THREAD

**Issue by bricesanchez** [rikkipitt/rails_admin_history_rollback#17]
I've recently updated my Rails Admin project to https://github.com/sferik/rails_admin/commit/53eef4fe2ec0953381f4e3197c885adc0423dd49 and i've this error when i try to start the server: 

``` bash
➜  *** git:(master) ✗ rs
/Users/bci20/Sites/***/.gems/gems/bundler-1.12.5/lib/bundler/runtime.rb:89:in `rescue in block (2 levels) in require': There was an error while trying to load the gem 'rails_admin_history_rollback'. (Bundler::GemRequireError)
Gem Load Error is: can't modify frozen Hash
Backtrace for gem load error is:
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback/extensions/paper_trail/auditing_adapter.rb:13:in `<class:AuditingAdapter>'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback/extensions/paper_trail/auditing_adapter.rb:12:in `<module:PaperTrail>'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback/extensions/paper_trail/auditing_adapter.rb:3:in `<module:Extensions>'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback/extensions/paper_trail/auditing_adapter.rb:2:in `<module:RailsAdmin>'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback/extensions/paper_trail/auditing_adapter.rb:1:in `<top (required)>'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback.rb:10:in `require'
/Users/bci20/Sites/***/vendor/ruby/2.3.0/gems/rails_admin_history_rollback-0.0.8/lib/rails_admin_history_rollback.rb:10:in `<top (required)>'
```

I think the problem comes from this commit:
https://github.com/sferik/rails_admin/commit/c94e36a983edd43d8fbbf6bd3243dee8c8844b19

How could we fix it ?

Note: I've rollback to rails_admin `v1.0.0.rc` to be able to start my project.
Note 2: I've finally rollback to rails_admin `~> 0.8.1` due to others bugs in the rc of Rails Admin.

**Comment by rikkipitt** [comment]
@bricesanchez Thanks for reporting, I'll take a look at this ASAP.

**Comment by maxexcloo** [comment]
Having this issue also!

**Comment by rikkipitt** [comment]
@bricesanchez and @maxexcloo RailsAdmin released v1.0.0 yesterday. I have found and fixed the problem locally, so I'll publish a new version of my gem today and let you know!

Thanks for reporting this guys, really appreciated.

**Comment by rikkipitt** [comment]
I've release v0.0.9 that solves this problem - update your gems and let me know if all is good your end!

Rikki
